src/reducers/versions.spec.tsx

```diff
@@ -121,6 +121,20 @@ describe(__filename, () => {
       );
     });
 
+    it('logs an error when updateSelectedPath is called for an unknown version', () => {
```

Comment:
The four test cases added that check that an error was logged for different actions are all essentially identical, other than the action being called, so I wonder if either a helper, or an `it.each()` would be appropriate for these to reduce the duplication in this file?